theme/templates/accounts/profile.html

```diff
@@ -157,15 +157,6 @@ <h4 class="text-muted">
                                 </div>
 
                                 <div class="row">
-                                    {# Username #}
-                                    <fieldset class="col-sm-4">
```

Comment:
If we are removing the 'username' from the form submission, it seems the ```update_user_profile()``` in theme/views.py will fail.